addons/spreadsheet/static/src/assets_backend/spreadsheet_binary_field/spreadsheet_binary_field.js

```diff
@@ -0,0 +1,21 @@
+import { registry } from "@web/core/registry";
```

Comment:
you tatally ingored/didn't see my comments on the old pr https://github.com/odoo/odoo/pull/180852 ...

Reply:
Sorry, when I sa your big comment on the flow in general, I did not scroll further and went straight to Francois to discuss the spec. Your comments were not ignored.